src/app/cli/src/graphql.ml

```diff
@@ -294,77 +302,65 @@ module Types = struct
             ~typ:(non_null @@ list @@ non_null user_command)
             ~args:Arg.[]
             ~resolve:(fun _ {user_commands; _} -> user_commands)
         ; field "feeTransfer"
+            ~doc:"List of fee transfers included in this block"
             ~typ:(non_null @@ list @@ non_null fee_transfer)
             ~args:Arg.[]
             ~resolve:(fun _ {fee_transfers; _} -> fee_transfers)
-        ; uint64_field "coinbase" ~doc:"Coinbase amount that proposer receives"
+        ; uint64_field "coinbase"
+            ~doc:"Number of tokens granted to the producer of this block"
             ~args:Arg.[]
             ~resolve:(fun _ {coinbase; _} -> Stringable.amount coinbase) ] )
 
-  let snark_fee : (Coda_lib.t, Transaction_snark_work.t option) typ =
-    obj "SnarkFee" ~fields:(fun _ ->
-        [ field "snarkCreator" ~typ:(non_null string)
-            ~doc:"public key of the snarker"
-            ~args:Arg.[]
-            ~resolve:(fun _ {Transaction_snark_work.prover; _} ->
-              Stringable.public_key prover )
-        ; field "fee" ~typ:(non_null string)
-            ~doc:"the cost of creating the proof"
-            ~args:Arg.[]
-            ~resolve:(fun _ {Transaction_snark_work.fee; _} ->
-              Currency.Fee.to_uint64 fee |> Unsigned.UInt64.to_string ) ] )
-
   let blockchain_state =
     obj "BlockchainState" ~fields:(fun _ ->
-        [ field "date" ~typ:(non_null string)
+        [ field "date" ~typ:(non_null string) ~doc:(Doc.date "date")
             ~args:Arg.[]
             ~resolve:(fun _ {Coda_state.Blockchain_state.Poly.timestamp; _} ->
               Block_time.to_string timestamp )
         ; field "snarkedLedgerHash" ~typ:(non_null string)
+            ~doc:"Base58Check-encoded hash of the snarked ledger"
             ~args:Arg.[]
             ~resolve:
               (fun _ {Coda_state.Blockchain_state.Poly.snarked_ledger_hash; _} ->
-              Frozen_ledger_hash.to_bytes snarked_ledger_hash )
+              Stringable.Frozen_ledger_hash.to_base58_check snarked_ledger_hash
+              )
         ; field "stagedLedgerHash" ~typ:(non_null string)
+            ~doc:"Base58Check-encoded hash of the staged ledger"
             ~args:Arg.[]
             ~resolve:
               (fun _ {Coda_state.Blockchain_state.Poly.staged_ledger_hash; _} ->
-              Ledger_hash.to_bytes
+              Stringable.Ledger_hash.to_base58_check
               @@ Staged_ledger_hash.ledger_hash staged_ledger_hash ) ] )
 
   let protocol_state =
     let open Filtered_external_transition.Protocol_state in
     obj "ProtocolState" ~fields:(fun _ ->
         [ field "previousStateHash" ~typ:(non_null string)
+            ~doc:"Base58Check-encoded hash of the previous state"
             ~args:Arg.[]
             ~resolve:(fun _ t ->
               Stringable.State_hash.to_base58_check t.previous_state_hash )
         ; field "blockchainState"
+            ~doc:"Base58Check-encoded hash of the blockchain state"
```

Comment:
is this current state?

Reply:
Not sure @bkase what should I say here, I got a little confused by all the hashes

Reply:
This is not a hash, it's the "blockchain state" as described above